source/include/ble/cs_BleCentral.h

```diff
@@ -0,0 +1,178 @@
+/*
+ * Author: Crownstone Team
+ * Copyright: Crownstone (https://crownstone.rocks)
+ * Date: Mar 26, 2021
+ * License: LGPLv3+, Apache License 2.0, and/or MIT (triple-licensed)
+ */
+
+#pragma once
+
+#include <ble/cs_Nordic.h>
+#include <ble/cs_UUID.h>
+#include <common/cs_Types.h>
+#include <events/cs_Event.h>
+#include <structs/cs_PacketsInternal.h>
+#include <events/cs_EventListener.h>
+
+/**
+ * Class that enables you to connect to a device, and perform write or read operations.
+ *
+ * Generally, you first connect, then discover, then perform read and/or write operations, and finally disconnect.
+ * Note that the disconnect event might happen at any time.
+ */
+class BleCentral: EventListener {
+public:
+	static BleCentral& getInstance() {
+		static BleCentral instance;
+		return instance;
+	}
+
+	void init();
```

Comment:
Can be removed:
Since `init()` only registers the listener, it is a duplicate of the function `EventListener::listen()`. 

(The `::listen()` method was introduced decouple the dependency on `EventDispatcher` for components that implement `EventListener`. That dependency is an implementation detail of `EventListener`.)

Reply:
Good suggestion, but should be done outside this PR.